notify/notify.go

```diff
@@ -80,29 +77,33 @@ func (n *Notifier) Init(options ...Option) error {
 func (n *Notifier) Send(ctx context.Context, skey int64, kind, msg string) error {
 	for _, f := range n.filters {
 		if !strings.Contains(msg, f) {
-			log.Printf("filter '%s' applied: not sending %s message to %s", f, kind, n.recipient)
+			log.Printf("filter '%s' applied: not sending %s message to %s", f, kind, n.Recipients())
 			return nil
 		}
 	}
 
 	if n.store != nil {
-		sendable, err := n.store.Sendable(ctx, skey, kind+"."+n.recipient)
+		sendable, err := n.store.Sendable(ctx, skey, kind+"."+n.Recipients())
 		if err != nil {
 			log.Printf("store.IsSendable returned error: %v", err)
 		}
 		if !sendable {
-			log.Printf("too soon to send %s a %s message", n.recipient, kind)
+			log.Printf("too soon to send %s message to %s", kind, n.Recipients())
 			return nil
 		}
 	}
 
-	log.Printf("sending %s a %s message", n.recipient, kind)
+	log.Printf("sending %s message to %s", kind, n.Recipients())
 
 	if n.publicKey != "" && n.privateKey != "" {
 		clt := mailjet.NewMailjetClient(n.publicKey, n.privateKey)
+		var recipients mailjet.RecipientsV31
+		for _, recipient := range n.recipients {
+			recipients = append(recipients, mailjet.RecipientV31{Email: recipient})
+		}
 		info := []mailjet.InfoMessagesV31{{
 			From:     &mailjet.RecipientV31{Email: n.sender},
-			To:       &mailjet.RecipientsV31{mailjet.RecipientV31{Email: n.recipient}},
+			To:       &recipients,
```

Comment:
This doesn't look quite right, why don't we still need to use the RecipientsV31 type here?

Reply:
RecipientsV31 is nothing more than []RecipientV31, which recipients is.